modules/orchestrator/ref-common-sonataflow-configuration-issues.adoc

```diff
@@ -0,0 +1,40 @@
+:_mod-docs-content-type: REFERENCE
+
+[id="ref-common-sonataflow-configuration-issues_{context}"]
+= Common SonataFlow configuration issues
+
+The following are common configuration problems encountered during SonataFlow deployment and direct solutions for verifying persistence and connectivity:
+
+. Workflow installed in a separate namespace than Sonataflow services fails to start.
++
+You can deploy a workflow to a namespace outside the core SonataFlow services (for example, `sonataflow-infra`). Use the following steps to enable persistence and connectivity for the workflow:
+
+Configure network policies for inter-namespace traffic::
+
+* Allow core SonataFlow services to receive traffic from the workflow namespace by creating a new `NetworkPolicy` or updating the existing one to include the new namespace.
+
+* Set up a separate `NetworkPolicy` in the workflow namespace to control communication, allowing traffic only between {product-very-short}, Knative, SonataFlow services, and workflows.
+
+* Optionally, create an `allow-intra-namespace` policy in the workflow namespace to enable unrestricted communication among all pods within that namespace.
+
+Ensure workflow persistence (If required)::
+
+* Create a dedicated PostgreSQL Secret containing database credentials within the workflow namespace.
+
+* Configure the workflow `serviceRef` property to correctly reference the PostgreSQL service namespace using the `namespace: $POSTGRESQL_NAMESPACE` attribute.
+
+Identify required namespaces::
+
+* Retrieve the namespace value where {product-very-short} is running using `oc get backstage -A`.
+
+* Identify the SonataFlow Services Namespace by checking for either a `sonataflowclusterplatform` or `sonataflowplatform` instance.
+
+. `sonataflow-platform-data-index-service` cannot connect to PostgreSQL database on startup.
++
+Diagnose database connection failures by performing the following checks:
+
+.. Make sure the PostgreSQL Pod has fully transitioned to a `running` and operational status.
```

Comment:
@gabriel-farache is there any external documentation we can link to with setps how to diagnose db issues?